preprocessors/caption.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/caption.schema.json",
+    "type": "object",
+    "title": "Caption generator",
+    "description": "Caption generated by ca.mcgill.a11y.image.preprocessor.captionGenerator",
+    "properties": {
```

Comment:
To confirm, are there no other properties that may be useful related to this type?

Reply:
nope. It is a single sentence description of the graphic. That's the only description generated by the preprocessor